azerothcore/azerothcore-wotlk
# fix(DB/loot): Unholy Alliance available without prerequisites

## Changes Proposed:
-  Make Unholy Alliance part 2 require completion of Unholy Alliance part 1

## Issues Addressed:

- Closes azerothcore#5984
- Closes chromiecraft/chromiecraft#678

## SOURCE:

- https://wowgaming.altervista.org/aowow/?quest=14352 This is the quest chain.

## Tests Performed:

- Went to Bragor Bloodfist in the Undercity to ensure https://wowgaming.altervista.org/aowow/?quest=14353 was not available.
- Added https://wowgaming.altervista.org/aowow/?item=49205 to my inventory and started part 1 of the quest series (https://wowgaming.altervista.org/aowow/?quest=14352)
- Handed in the quest to Bragor Bloodfist and received part 2 of the quest series

## How to Test the Changes:


1. Run ```SET @PART1 = 14352;
SET @PART2 = 14353;
SELECT * FROM `acore_world`.`quest_template_addon` WHERE ID IN (@PART1, @PART2);``` 
For ID 14352 NextQuestID should be 14353. 
For ID 14353 PrevQuestID should be 14352
2. Go to Bragor Bloodfist in Undercity and make sure you can't get the quest before completing part 1.
3. Add https://wowgaming.altervista.org/aowow/?item=49205 to inventory in order to get part 1 of the quest and complete it by talking to Bragor Bloodfist.
4. You should not be able to accept part 2 of the quest series.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
